kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/java/org/kie/workbench/common/stunner/client/widgets/inlineeditor/InlineTextEditorBoxViewImpl.java

```diff
@@ -136,7 +138,16 @@ public void show(final String name, final double width, final double height) {
         nameField.setAttribute("data-text", placeholder);
 
         setVisible();
-        scheduleDeferredCommand(() -> nameField.focus());
+        scheduleDeferredCommand(() -> {
+            nameField.focus();
+            selectText(nameField);
+        });
+    }
+
+    public void selectText(Div node) {
+        if (DomGlobal.window != null) {
```

Comment:
curious, when can this happen? When can `DomGlobal.window` be null?

Reply:
Hi @inodeman , it think it's also good tyring to drop that null check fo the Window object, if possible. You should run the tests by using the `@LienzoMockitoTestRunner`, this way I'm pretty sure the window will not be null. Are you running tests cases using this runner? Also otherwise check other tests what they're doing with the window object, but I think when using the lienzo runner the instance is being set by default.

Reply:
+1 checks like this just for testing purposes are not a way to go.